package.json

```
@@ -89,36 +80,49 @@
     "eslint-plugin-storybook": "^0.5.12",
     "eslint-plugin-unused-imports": "^2.0.0",
     "file-loader": "^6.1.1",
+    "highlight.js": "^11.3.1",
     "html-loader": "^1.3.1",
     "html-webpack-plugin": "^4.5.0",
     "husky": "7.0.4",
     "lint-staged": "^12.4.1",
-    "mini-css-extract-plugin": "0.9.0",
+    "lodash.isplainobject": "^4.0.6",
+    "lowlight": "^1.20.0",
     "postcss": "^8.4.14",
     "postcss-import": "^14.1.0",
     "postcss-loader": "^4.0.3",
     "postcss-preset-env": "^6.7.0",
     "prettier": "^2.6.1",
     "prettier-plugin-tailwindcss": "^0.1.8",
+    "ramda": "^0.28.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-hotkeys-hook": "^3.4.4",
+    "react-masonry-infinite": "^1.2.2",
+    "react-popper": "^2.2.5",
     "react-scripts": "4.0.3",
+    "rollup": "^2.76.0",
+    "rollup-plugin-analyzer": "^4.0.0",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "rollup-plugin-styles": "^4.0.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "rollup-plugin-visualizer": "^5.6.0",
     "sass": "^1.26.11",
     "sass-loader": "^10.0.2",
     "style-loader": "^2.0.0",
     "tailwindcss": "npm:@tailwindcss/postcss7-compat",
+    "tippy.js": "^6.3.1",
     "webpack": "5.64.3",
     "webpack-cli": "4.9.1",
     "webpack-dev-server": "4.7.4"
   },
   "peerDependencies": {
+    "highlight.js": "^11.3.1",
     "react": "17.0.2",
     "react-dom": "17.0.2"
   },
```

I think we can move a few more `dependencies` to `peerDependencies`.  These packages are always present in the parent project.
```suggestion
  "peerDependencies": {
    "@bigbinary/neeto-icons": "^1.8.25",
    "axios": "^0.24.0",
    "classnames": "^2.3.1",
    "highlight.js": "^11.3.1",
    "react": "17.0.2",
    "react-dom": "17.0.2"
  },
```

---

We've to consider neetoEditor is an open-source package. We shouldn't be making any changes with the idea of using this package only in neeto environment.

I was considering replacing axios with fetch, as there are only a few use cases.